tests/test_common.py

```diff
@@ -14,25 +14,60 @@
 LIST3 = [1, 2, 3, [4, 5, 6, [7, 8, 9]]]
 
 
-@pytest.mark.parametrize("operation", ["invoke", "call"])
+@pytest.mark.only
+@pytest.mark.parametrize(
+    "operation, signature, max_fee, query_flag, mainnet_token",
+    [
+        ("invoke", None, 0, "simulate", None),
+        ("call", None, None, None, None),
+        ("deploy", None, 5, None, "token"),
+        ("declare", None, 3, "estimate_fee", "token2"),
+    ],
+)
```

Comment:
have you considered doing all permutations? or is it too slow?
```suggestion
@pytest.mark.parametrize('operation', ['invoke', 'call', 'deploy', 'declare'])
@pytest.mark.parametrize('signature', [None])
...
```

Reply:
I think that's a great idea! If I add this, it won't be too slow, but I'm not sure what will happen if we adopt this for all the tests. If this makes the test suite too slow in the future, we can refactor this then, I guess. I will add it.